pkg/controller/worker/machines.go

```diff
@@ -88,6 +92,9 @@ func (w *workerDelegate) generateMachineConfig(_ context.Context) error {
 		machineClasses     []map[string]interface{}
 		machineImages      []apisgcp.MachineImage
 	)
+	const (
+		csiMigrationVersion = "1.21"
```

Comment:
Gardener has already dropped support for anything bellow 1.20 so the check is not necessary. Regardless the CSI migration version for gcp was 1.18 if I am not mistaken.

Reply:
Ok, was un-certain about this. Will remove the check for CSI. thanks.